### PR TITLE
Create red4sec summary file

### PR DESCRIPTION
**Pull request by jackalyst**
Allows us to provide 1 link (ie. https://theqrl.org/theQRL/audits/) for those wanting to see our audits instead of one link going to audits and another going to an ad hoc medium post.

Suggestion credit goes to discord user Anon-and-mous, cheers

### CODE REVIEW

File: red4sec audit summary.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Red4Sec audit summary
+
+- Audit length: 7 weeks. 
+- Auditor homepage: https://red4sec.com
+
+Red4Sec’s report was exhaustive, and covered the entirety of our project space, with the exception of our post-quantum cryptography, which was coverd by x41 D-Sec. Below is a rundown of issues raised by Red4Sec by section of our network, as well as impact level.
```
Comment (jplomas) [inline red4sec audit summary.md]: “Coverd” typo could become “reviewed separately”

File: red4sec audit summary.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+During the analysis, whose report runs over 200 pages, a total of 67 vulnerabilities were detected. Many of these vulnerabilities did not pose any risk by themselves and therefore have been classified as informative.
+
+All vulnerabilities were classified according to the impact level defined by CVSS (Common Vulnerability Scoring System): critical, high, medium, low and informative.
+
+A summary of the issues per project area is described below.
+
+## Blockchain
+
+### QRL (Python)
```
Comment (jplomas) [inline red4sec audit summary.md]: ### QRL Node (Python)

**Review by jplomas — CHANGES_REQUESTED** [review]

File: red4sec audit summary.md (hunk 1 above)
Comment (jackalyst) [reply red4sec audit summary.md]: Good catch and suggestion. Change made.

File: red4sec audit summary.md (hunk 2 above)
Comment (jackalyst) [reply red4sec audit summary.md]: Thank you. Change made.

**Review by jplomas — APPROVED** [review]
👍

**Pull request merged**